.github/workflows/build-common.yml

```diff
@@ -44,8 +44,6 @@ jobs:
         uses: gradle/actions/setup-gradle@94baf225fe0a508e581a564467443d0e2379123b # v4.3.0
         with:
           cache-read-only: ${{ inputs.cache-read-only }}
-          # gradle enterprise is used for the build cache
-          gradle-home-cache-excludes: caches/build-cache-1
```

Comment:
removing this might help CI performance now that we aren't using gradle enterprise anymore (it should use github actions cache)